### ISSUE TITLE
Certificate Error

### ISSUE THREAD

**Issue by averagenokiauser** [artem78/s60-maps#57]
Certificate Error. Contact Application Supplier

I get this error does anyone know how to fix this?

**Comment by fizolas** [comment]
It sounds like you need to root (i.e. hack) your phone.

**Comment by theperson50** [comment]
Yes, that message means that the application requires a permission that even self-signing cannot grant, meaning that either you need to sign the app with a developer certificate or hack the phone
